docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -1,12 +1,22 @@
 ---
-title: Helm Charts in Rancher
+title: Helm Charts and Apps
 ---
 
 <head>
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/helm-charts-in-rancher"/>
 </head>
 
-In this section, you'll learn how to manage Helm chart repositories and applications in Rancher. Helm chart repositories are managed using **Apps**. It uses a catalog-like system to import bundles of charts from repositories and then uses those charts to either deploy custom Helm applications or Rancher's tools such as Monitoring or Istio. Rancher tools come as pre-loaded repositories which deploy as standalone Helm charts. Any additional repositories are only added to the current cluster.
+In this section, you'll learn how to manage Helm chart repositories and apps in Rancher. 
+
+Helm chart repositories in Rancher are managed using **Apps**.
+
+## Catalogs
```

Comment:
I agree on reducing the header size down to a H3. I feel like it also has to be moved, otherwise it would look messy, since the next section is an H2. "Catalog system" I'm not a fan of, since  the newer system is still referred to as catalog-like. It's tricky because we want searches for "catalogs" to hit on this section header, but the header still needs to make sense in context with the rest of the page.

Reply:
For sure makes sense! Maybe since a little below there is a header with "Versioning Scheme" and "Charts", something like "Catalogs Naming Scheme" could go in between these header's and a description there?

Reply:
@sunilarjun Updated this section and header in https://github.com/rancher/rancher-docs/pull/1170/commits/7e8be392903254180191b3c24497706938c35748

Reply:
Header looks good to go, thanks for updating Marty! Approving now